### DISCUSSION TITLE
How to connect API for multiple account

### DISCUSSION THREAD

**Discussion by jeonghoHyeon-kontactic** [amzn/ads-advanced-tools-docs#23] [Q&A] [answered]
On Seller Central, I have more than 10 accounts user permission either as admin or View & Edit  
I'd like to bring every data regarding Ads just as Campaign Reports, Search Term Reports, etc. 
As I understand Amazon document to connect API, the following step is below

1) Sign in Developer 
2) Create Security Profile
3) Join Amazon Ad Partner Network 
4) Requet Amazon Ads API Access on Amazon Ad Partner Network 

But the thing is that there is no explanation about requesting API for multiple accounts.
More specifically, there is no question which account API I'd like to call on the application of Amazon API Access for Step 4 
Then I wonder whether API has every data for multiple accounts once Amazon Ads API Access requested is approved

**Comment by jerirose** [comment, marked as answer]
API access is granted to a _client application_ and represents permission simply to call the API. 

A single client application can access data and services for multiple advertiser accounts, with explicit permission from those accounts. The specific advertiser account being accessed in a given API request is determined by an _access token_ passed in the request headers, so your client application will use a different access token to access each advertiser account.

The process for retrieving access tokens is described in the ["Getting started" section of the advanced tools center](https://advertising.amazon.com/API/docs/en-us/getting-started/overview). You can also find more conceptual information about accounts and identity in the ["Authorization" section](https://advertising.amazon.com/API/docs/en-us/concepts/authorization/overview).

**Comment by jerirose** [comment]
Another note: Joining the Partner Network is not required for accessing the API. You can also [apply for API access as a direct advertiser](https://advertising.amazon.com/API/docs/en-us/onboarding/apply-for-access#to-apply-for-api-access-as-a-direct-advertiser).

See [more information about the Partner Network here](https://advertising.amazon.com/partners/network) to determine whether it's right for your use case.

**Reply by Ss110525** [reply]
We have more than 10 accounts on Amazon in the United States. Can we also apply for API access as a direct advertiser to manage advertisements for all accounts?

**Reply by jerirose** [reply]
Hi @Ss110525 , API access is limited to [one client application per company](https://advertising.amazon.com/API/docs/en-us/guides/onboarding/create-lwa-app#retrieve-your-security-credentials), but there is no limit to the number of advertising accounts that can be managed by a single client application. 

The client application makes requests on behalf of a user account determined by the _access token_ used in the request to the API. By maintaining access tokens for all accounts to which you have access, you can use a single approved client application to manage any number of accounts.

You may also choose to explore *manager accounts* as a way of consolidating access to multiple advertising accounts. To learn more about manager accounts, see:
- [Manager accounts in the Amazon Ads API](https://advertising.amazon.com/API/docs/en-us/guides/account-management/authorization/manager-accounts)
- [Manager accounts in the advertising console](https://advertising.amazon.com/help/GU3YDB26FR7XT3C8)

**Comment by SylvieManx** [comment]
可以的

Ss110525 ***@***.***> 于2024年4月7日周日 16:58写道：
